pkg/cmd/prebuild/add.go

```diff
@@ -134,8 +181,10 @@ var prebuildAddCmd = &cobra.Command{
 	},
 }
 
-var runOnAddFlag bool
-
 func init() {
 	prebuildAddCmd.Flags().BoolVar(&runOnAddFlag, "run", false, "Run the prebuild once after adding it")
+	prebuildAddCmd.Flags().StringVarP(&branchFlag, "branch", "b", "", "Git branch for the prebuild")
+	prebuildAddCmd.Flags().IntVarP(&retentionFlag, "retention", "r", 0, "Retention period for the prebuild (in days)")
+	prebuildAddCmd.Flags().IntVarP(&commitIntervalFlag, "commit-interval", "c", 0, "Commit interval for the prebuild (in seconds)")
```

Comment:
The description for commit interval should be: "Commit interval for running a prebuild - leave blank to ignore push events"

Reply:
updated